authentication/scheme.js

```diff
@@ -2,7 +2,7 @@
  * Authentication scheme used to authenticate requests to publish.webmaker.org
 */
 
-var Boom = require('boom');
+// var Boom = require('boom');
```

Comment:
Right? This file uses `boom` in its example code below, so I figured I'd leave this to make it clear that it's a dependency.